cmd/harmony/main.go

```diff
@@ -416,6 +425,27 @@ func setupConsensusAndNode(nodeConfig *nodeconfig.ConfigType) *node.Node {
 	return currentNode
 }
 
+func setupBlacklist() (*map[ethCommon.Address]bool, error) {
+	if _, err := os.Stat(*blacklistPath); os.IsNotExist(err) {
+		return nil, errors.New(fmt.Sprintf("blacklist file not found at `%s`", *blacklistPath))
+	}
+	dat, err := ioutil.ReadFile(*blacklistPath)
+	if err != nil {
+		return nil, err
+	}
+	addrMap := make(map[ethCommon.Address]bool)
+	for _, line := range strings.Split(string(dat), "\n") {
+		if len(line) != 0 { // blacklist file may have trailing empty string line
+			addr, err := common.Bech32ToAddress(line)
+			if err != nil {
+				return nil, err
```

Comment:
you can also just continue and skip the line with error

Reply:
I prefer all or nothing when loading addresses.

Reply:
I prefer all or nothing